src/logic/transaction.js

```diff
@@ -794,6 +1039,32 @@ Transaction.prototype.verifyUnconfirmed = ({ trs, sender, cb }) => {
     });
 };
 
+Transaction.prototype.newVerifyUnconfirmed = ({ trs, sender }) => {
+    const calculateFee = __private.types[trs.type].calculateUnconfirmedFee || __private.types[trs.type].calculateFee;
+    const fee = calculateFee.call(self, trs, sender) || 0;
+    if (trs.type !== transactionTypes.REFERRAL &&
+        !(trs.type === transactionTypes.STAKE && trs.stakedAmount < 0) &&
```

Comment:
`!(trs.type === transactionTypes.STAKE && trs.stakedAmount < 0)` logic must be in verify transaction fields